internal/mode/static/nginx/config/upstreams.go

```diff
@@ -129,8 +130,10 @@ func (g GeneratorImpl) createUpstream(up dataplane.Upstream) http.Upstream {
 		}
 	}
 
+	// update the upstream name to include the "upstream_" prefix,
+	// to match the backend name in the proxy_pass directive.
```

Comment:
should we add this comment in `split_clients.go` and here? felt kind of repeated in both the files.